source/_integrations/light.markdown

```diff
@@ -12,7 +12,7 @@ This integration allows you to track and control various light bulbs. Read the i
 
 ### Default turn-on values
 
-To set the default color and brightness values when the light is turned on, create a custom `light_profiles.csv` (as described below in the `profile` attribute of `light.turn_on`).
+To set the default color, brightness and transition values when the light is turned on, create a custom `light_profiles.csv` (as described below in the `profile` attribute of `light.turn_on`).
```

Comment:
@frenck Not a fan of the [oxford comma](https://andrewpegoda.com/2014/06/24/the-oxford-comma-plus-every-comma-rule-you-need-to-know/), I take it! 😝

Reply:
It is not about me being a fan or not, it is about it is not according to the project documentation standards. <https://developers.home-assistant.io/docs/documenting/standards>

Which are merely in place to keep things standardized.

Reply:
Doh! My bad. I even read that page last month when I started contributing and promptly forgot that detail. Apologies for the ribbing :)